README.md

```diff
@@ -38,7 +42,7 @@ We'd like you to consider how your code could be recomposable into new solutions
 
 ## Project
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app). It provides a base point for starting the test with some services in `./src/services/CAPI.js` to get the data you need from CAPI (the Guardian's Content API) and an `App` component that uses these services to fetch data and render it all of the responses to the DOM. You should not need to access any other properties on the CAPI response than those already accessed in the project already (`webTitle` and `webUrl`).
```

Comment:
```suggestion
This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app). It provides a base point for starting the test with some services in `./src/services/CAPI.js` to get the data you need from CAPI (the Guardian's Content API) and an `App` component that uses these services to fetch data and render all of the responses to the DOM. You should not need to access any other properties on the CAPI response than those already accessed in the project already (`webTitle` and `webUrl`).
```

Reply:
Wow that does not look like what I expected. Basically, there's an `it` we can remove: render ~it~ all.

Reply:
Good eyes 👀